include/CLI/Option.hpp

```diff
@@ -517,6 +517,9 @@ class Option : public OptionBase<Option> {
     /// Get the description
     const std::string &get_description() const { return description_; }
 
+    /// Set the description
+    void description(const std::string& description) { description_ = description; }
```

Comment:
Could you please return Option* (`this`)? That way these can be chained.

Reply:
Don't worry about style, but if you are editing it anyway, `std::string &description` will make clang-format happy.

Reply:
Np, I'll fix!